### ISSUE TITLE
[Feature Request] Protocol columns

### ISSUE THREAD

**Issue by HLWeil** [nfdi4plants/Swate#207]
# Protocol Type column: 

Motivation: https://github.com/nfdi4plants/nfdi4plants_ontology/issues/32

To fill out the `protocolType` field in the `protocol.json`, it would be nice to have a `Protocol Type` column in `SWATE`. This could be insertable into the table the same way as Input or Output columns.
And the `Ontology term search` feature could be used to insert ontology terms related to the `Protocol Type` in our ontology.

This might look something like this:
![image](https://user-images.githubusercontent.com/17880410/176671525-f6dbc2a9-f5fd-4d98-a817-a65925855c1d.png)

# Protocol REF column: 

Currently, the Excel Sheet name is used as the protocol name. The problem is, that the excel sheet names are restricted to 31 characters. To allow for longer protocol names and also to give more control to the user, the `Protocol REF` column could be introduced in SWATE . This column is already defined in the `ISA-Tab` standard.
It could also be inserted into the table like the input and output columns.

**Comment by Freymaurer** [comment]
According Part 1 of this issue, we now have featured columns. I decided on this name, because it more or less is "Parameter [protocol type]" made easier accessible for the user. They can be found by clicking on the star symbol, which will open a new subpage in the dropdown.

I chose this design to keep the main dropdown as clean as possible, while also allowing for more future additions like "Protocol Type"

![image](https://user-images.githubusercontent.com/39732517/177402067-8a793137-6e38-4e9d-82f8-a4314380ea83.png)
![image](https://user-images.githubusercontent.com/39732517/177402233-011043f5-d05b-4ebf-a628-4fdc4515de5a.png)
